generators/entity/prompts.js

```diff
@@ -534,6 +534,9 @@ function askForField(done) {
                 } else if (jhiCore.isReservedKeyword(input, 'JAVA')) {
                     return 'Your enum name cannot contain a Java reserved keyword';
                 }
+                if (!/^[A-Za-z0-9_\s]*$/.test(input)) {
```

Comment:
Why whitespaces?

Reply:
Copied it from below, thanks for catching that